modules/programs/khal-accounts.nix

```diff
@@ -13,5 +13,39 @@ with lib;
         Keep khal from making any changes to this account.
       '';
     };
+
+    color = mkOption {
+      type = types.nullOr (types.enum [
+        "black"
+        "white"
+        "brown"
+        "yellow"
+        "dark gray"
+        "dark green"
+        "dark blue"
+        "light gray"
+        "light green"
+        "light blue"
+        "dark magenta"
+        "dark cyan"
+        "dark red"
+        "light magenta"
+        "light cyan"
+        "light red"
+      ]);
+      default = null;
+      description = ''
+        Color in which events in this calendar are displayed.
+      '';
+      example = "light green";
+    };
+
+    priority = mkOption {
+      type = types.int;
+      default = 10;
+      description = ''
+        Priority of a calendar used for coloring.
```

Comment:
could you mention which is highest priority, eg.
`(priority 0 is shown before priority 10)`

Reply:
Fixed in f6c4426.